docs/workflow/requirements/macos-requirements.md

```diff
@@ -10,6 +10,11 @@ These instructions were validated on macOS 10.13 High Sierra.
 
 If your machine has Command Line Tools for XCode 6.3 installed, you'll need to update them to the 6.3.1 version or higher in order to successfully build. There was an issue with the headers that shipped with version 6.3 that was subsequently fixed in 6.3.1.
 
+Xcode
+-----
+
+Install Apple Xcode developer tools from the Mac App Store ([link](https://apps.apple.com/us/app/xcode/id497799835)).
```

Comment:
The previous paragraph talks about Command Line Tools for XCode already. Should those two be folded together?

Reply:
I think we should just delete that line (about Xcode 6.3.1); it has been there since 2015, and the current version of Xcode is 11.3.

Reply:
Agree.